Overwatch/style.scss

```diff
@@ -0,0 +1,39 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,700;0,800;1,400&display=swap');
+@import url(navbar.css);
+@import url(bg-1.css);
```

Comment:
Mettiamo i file inclusi in una folder.. Così li ritroviamo più facilmente...

Reply:
Ho fatto come hai richiesto